man/docker-create.1.md

```diff
@@ -148,6 +149,15 @@ two memory nodes.
                                'container:<name|id>': reuses another container shared memory, semaphores and message queues
                                'host': use the host shared memory,semaphores and message queues inside the container.  Note: the host mode gives the container full access to local shared memory and is therefore considered insecure.
 
+**--kernel-memory**=""
+   Kernel memory limit (format: <number><optional unit>, where unit = b, k, m or g)
+
+   Allows you to constrain the kernel memory available to a container. If a
```

Comment:
Constrains the kernel memory available to a container. If a limit of `0` is specified (not using `--kernel-memory`), the container's kernel memory is not limited. If you specify a limit,  it may be rounded up to a multiple of the operating system's page size and the value can be very large, millions of trillions.